### ISSUE TITLE
low volume on playback on Raspberry Pi

### ISSUE THREAD

**Issue by DavidLaidlaw** [andrewrk/groovebasin#443]
I am running groove basin on a raspberry pi.  Playback volume is low enough that, even at full volume on the amp it's connected to and in groove basin, it's not very loud.  Audio connected to the same amp from an iPhone is fine.

Not sure what other info would be helpful to provide, so please request!

**Comment by andrewrk** [comment]
Just double checking, you have found this volume slider, right?

![](http://i.imgur.com/uOof8Ah.png)

Also, could you double check the volume on your system? Does playing audio with a different application than Groove Basin sound loud enough?

**Comment by DavidLaidlaw** [comment]
Yes, that slider in GrooveBasin is at 100% (I didn’t go higher).

Not sure what else to use to generate audio from the RPi.  Any suggestions?

Thanks!

-David

--
David Laidlaw, Professor, Brown Computer Science
Box 1910, Providence, RI 02912, CIT 521, +1-401-354-2819
http://www.cs.brown.edu/~dhl

**Comment by andrewrk** [comment]
You could install sox and use the command line "play" command.

If the volume is too quiet, increase that volume slider to beyond 100%. The warning that it gives you about audio quality is real; the reason Groove Basin is quieter by default is to create headroom for loudness normalization across multiple tracks. So when you increase that volume slider, you are adjusting the balance between audio quality and player loudness.

**Comment by DavidLaidlaw** [comment]
If the output signal is really too low, I’d rather get a preamp and put it between the RPi and the amp.  That would be a better quality solution.  But if there is some other volume setting that I’m missing, then it might be better to tweak that rather than amplifying a too-low analog signal…

I’ll see what “sox” does.

Thanks!

-David

--
David Laidlaw, Professor, Brown Computer Science
Box 1910, Providence, RI 02912, CIT 521, +1-401-354-2819
http://www.cs.brown.edu/~dhl

**Comment by andrewrk** [comment]
Sadly the analog output on the rpi, in my experience, has a humming sound. It would be nice to figure out a way to use the HDMI output for audio so that no on-board interference could mangle the sound.

**Comment by DavidLaidlaw** [comment]
Interesting.  I hadn’t noticed that.  That can sometimes be a ground issue.  Might see if decoupling some of your components from your AC ground helps?

Also, I learned recently that the sound in HDMI is compressed digital.  So you may not be able to get what you want that way, either…

-d

--
David Laidlaw, Professor, Brown Computer Science
Box 1910, Providence, RI 02912, CIT 521, +1-401-354-2819
http://www.cs.brown.edu/~dhl

**Comment by norpol** [comment]
Since it haven't been suggested: Have you just tried to open ['alsamixer'](https://en.wikipedia.org/wiki/Alsamixer) on the command-line of the raspberrypi? You can increase the volume there.
